examples/platform/alerts_common_scenarios.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env python
+# *******************************************************
+# Copyright (c) VMware, Inc. 2020-2023. All Rights Reserved.
+# SPDX-License-Identifier: MIT
+# *******************************************************
+# *
+# * DISCLAIMER. THIS PROGRAM IS PROVIDED TO YOU "AS IS" WITHOUT
+# * WARRANTIES OR CONDITIONS OF ANY KIND, WHETHER ORAL OR WRITTEN,
+# * EXPRESS OR IMPLIED. THE AUTHOR SPECIFICALLY DISCLAIMS ANY IMPLIED
+# * WARRANTIES OR CONDITIONS OF MERCHANTABILITY, SATISFACTORY QUALITY,
+# * NON-INFRINGEMENT AND FITNESS FOR A PARTICULAR PURPOSE.
+
+"""
+This example shows how to use the Alerts API and relate Alerts to other objects such as Processes and Observations.
+
+The SDK documentation is published on Read The Docs.  An Alerts Guide is available there.
+https://carbon-black-cloud-python-sdk.readthedocs.io
+
+If you are using SDK version 1.4.3 or earlier, see the related example script, alert_v6_v7_migration.py,
+also in examples/platform and the Alert Migration Guide on Read the Docs on the changes that are needed.
+SDK v1.5.0 does have breaking changes.
+"""
+
+import sys
+import time
+import json
+from cbc_sdk import CBCloudAPI
+from cbc_sdk.platform import Alert, WatchlistAlert
+from cbc_sdk.platform import Device
+
+# To see the http requests being made, and the structure of the search requests enable debug logging
+# import logging
+# logging.basicConfig(level=logging.DEBUG)
+
+
+def alert_workflow(api):
+    """The workflow was simplified in SDK 1.5.0 to align with Alert v7 API.
+
+    1. Use an Alert Search to specify the alerts that will have their status updated
+
+    * The request body is a search request and all alerts matching the request will be updated.
+    * Two common uses are to update one alert, or to update all alerts with a specific threat id.
+    * Any search request can be used as the criteria to select alerts to update the alert status.
+
+    2. Submit a job to update the status of Alerts.
+
+    * The status can be ``OPEN``, ``IN PROGRESS`` or ``CLOSED`` (previously ``DISMISSED``).
+    * A Closure Reason may be included.
+
+    3. The immediate response confirms the job was successfully submitted.
+
+    4. Use the :py:mod:`Job() cbc_sdk.platform.jobs.Job` class to determine when the update is complete.
+
+    * Use job.await_completion().result()
+
+    5. Refresh the Alert Search to get the updated alert data into the SDK.
+
+    6. The future Alerts with the same threat id can be set to automatically close.
+    """
+    # This example closes a single alert.  Any alert search can be used.
+    ALERT_ID = "4ae2e0a4-3115-4692-8452-2ecd71db36ab"
```

Comment:
Maybe to get a random Alert with select(Alert), because this will stop working once this alert is not present anymore.